apps/sv/src/main/scala/org/lfdecentralizedtrust/splice/sv/migration/DomainMigrationDump.scala

```diff
@@ -105,4 +105,37 @@ object DomainMigrationDump {
     )
   }
 
+  def getDomainMigrationDumpUnsafe(
```

Comment:
how is this different from the one used for DR? I think I don't understand why we need another one here.